.teamcity/_self/projects/WPComTests.kt

```diff
@@ -357,8 +357,21 @@ fun jetpackPlaywrightBuildType( targetDevice: String, buildUuid: String): E2EBui
 				allowEmpty = false
 			)
 			param("env.VIEWPORT_NAME", "$targetDevice")
+			param("env.TARGET_JETPACK", "1")
```

Comment:
Ack, two small things I missed here!

1. The name should match the value in `envVariables`, so `TEST_ON_JETPACK`.
2. It seems for booleans we actually use `"true"` or `"false"` instead of `1`/`0`.

Reply:
Good catch, thanks!

Reply:
Changed to a checkbox type with default value of `true` for the Jetpack build.